test/libscap/linux/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+add_executable(unit-test-libscap-linux test_fds.c)
+
+target_link_libraries(unit-test-libscap-linux "${GTEST_LIB}" "${GTEST_MAIN_LIB}" scap)
+
+add_test(NAME unit-test-libscap-linux COMMAND unit-test-libscap-linux)
```

Comment:
Who is going to run `unit-test-libscap-linux` ? Can we somehow link it to `libscap_test` to make it automatically run (but only on linux of course)?

Reply:
Oh they are already part of the libscap_test sources, sorry. So, this is just a small helper to only run linux tests?

Reply:
I added this when I first gave up on implementing the test fully in C++, before I decided on scap_fds_impl.c. It's not necessary and I've removed it in https://github.com/falcosecurity/libs/pull/2271/commits/2c250970606a0732728aed755db1ad1d89f7c6e4.